fvm/scriptEnv.go

```diff
@@ -439,7 +443,16 @@ func (e *ScriptEnv) meterComputation(kind, intensity uint) error {
 }
 
 func (e *ScriptEnv) MeterComputation(kind common.ComputationKind, intensity uint) error {
-	return e.meterComputation(uint(kind), intensity)
+	select {
```

Comment:
Should this change be in the private `meterComputation` receiver instead? Otherwise, calling `meterComputation` will go around this logic. 

Another question is if `e.sth.EnforceComputationLimits` is disabled, should we check if the context is done. Usually `e.sth.EnforceComputationLimits` is disabled for parts that should not be interrupted, though that is not critical in scripts.